backend/src/v1/routes/report-routes.ts

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import passport from 'passport';
+import { auth } from '../services/auth-service';
+import { reportService } from '../services/report-service';
+import HttpStatus from 'http-status-codes';
+import { utils } from '../services/utils-service';
+
+const reportRouter = express.Router();
+
+/**
+ * /v1/report/published
+ *
+ * Get all published reports for the company associated with the logged in user.
+ */
+reportRouter.get(
+  '/published',
```

Comment:
change the url - remove /published , add a` query parameter` called `report_status` (which can be take values either `published` or `draft`)